server/config/topic_page/climate/USA.textproto

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## This file specifies the topic page config for climate in USA
+
+metadata {
+  topic_id: 'climate'
+  topic_name: 'Climate'
+  place_dcid: 'country/USA'
+  place_dcid: 'geoId/06'
```

Comment:
I noticed in the design doc that there is an additional field in metadata for a map of contained_place_types. Is that something we're still going to include? I think it's something we'll need for a few of the tiles like map and scatter

Reply:
yes! that's what it was meant for and we should absolutely include it. there are a few other issues to debug with the configs, and i was hoping to keep that separate from this.